specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2020-03-01/iotcentral.json

```diff
@@ -0,0 +1,868 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-09-01",
+    "title": "IotCentralClient",
+    "description": "Use this API to manage IoT Central Applications in your Azure subscription."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.IoTCentral/IoTApps/{resourceName}": {
+      "get": {
+        "tags": [
+          "Apps"
+        ],
+        "description": "Get the metadata of an IoT Central application.",
+        "operationId": "Apps_Get",
+        "x-ms-examples": {
+          "Apps_Get": {
+            "$ref": "./examples/Apps_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The body contains all the non-security properties of the IoT Central application. Security-related properties are set to null.",
+            "schema": {
+              "$ref": "#/definitions/App"
+            }
+          },
+          "default": {
+            "description": "Default error response",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "deprecated": false
```

Comment:
the default value should be false. well. This is correct but not commonly appear. 😄